docs/guides/using-azure-artifacts.md

```diff
@@ -0,0 +1,30 @@
+# Using Azure Artifacts
+
+`uv` can install packages from [Azure DevOps Artifacts](https://learn.microsoft.com/en-us/azure/devops/artifacts/start-using-azure-artifacts?view=azure-devops&tabs=nuget%2Cnugetserver). You can authenticate to a feed using a [Personal Access Token](https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=Windows) or interactively using Keyring.
+## Authenticate using a PAT
+If you have a Personal Access Token available (eg [`$(System.AccessToken)` in an Azure pipeline](https://learn.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#systemaccesstoken)), you can authenticate to Artifacts using Basic Auth. Simply include the PAT in the password field of the URL. The username can be any dummy string.
+
+```bash
+# if your token is in the ADO_PAT environment variable
+export UV_EXTRA_INDEX_URL=https://dummy:$ADO_PAT@pkgs.dev.azure.com/{organisation}/{project}/_packaging/{feedName}/pypi/simple/
+
+uv pip install my-private-package
+```
```

Comment:
Haven’t tested that workflow myself and I don’t know how common it would be (seems to me that you’d either use the plugin, if you have the wherewithal to preinstall stuff, or not use Keyring at all?). I can certainly give it a go tomorrow and add a note if you think it’d be worthwhile. Let me know